src/everett/ext/inifile.py

```diff
@@ -145,7 +145,7 @@ def __init__(self, possible_paths: Union[str, List[str]]) -> None:
                 break
 
         if not self.path:
-            logger.debug("No INI file found: %s", possible_paths)
+            logger.debug(f"No INI file found: {possible_paths}")
```

Comment:
You don't want to do this because it forces interpolation to always happen rather than letting the logger do it only if it's going to emit the message.